bench/bench.hs

```diff
@@ -84,25 +84,28 @@ runBCTest :: BCTests.Case -> IO Bool
 runBCTest x =
  do
   let vm0 = BCTests.vmForCase x
-  result <- execStateT (Stepper.interpret (Fetch.zero 0 (Just 0)) . void $ Stepper.execFully) vm0
-  maybeReason <- BCTests.checkExpectation False x result
-  pure $ isNothing maybeReason
+  result <- Stepper.interpret (Fetch.zero 0 Nothing) vm0 Stepper.execFully
+  case result of
+    Left _ -> pure False
+    Right _ -> pure True
+  -- maybeReason <- BCTests.checkExpectation False x (_ result)
```

Comment:
is this no longer needed? can we remove commented lines?

Reply:
I was not sure if this is a good replacement. @d-xo what do you think?

Reply:
I don't think this is a good replacement. `checkExpectation` is comparing various parts of the state to the expected results defined in the blockchain tests, this new code is just checking if the execution was succesful or not....